test/slax_web/websockets/issue_test.exs

```diff
@@ -0,0 +1,53 @@
+defmodule SlaxWeb.Issue.Test do
+  use SlaxWeb.ConnCase, async: true
+  import Mox
+
+  alias SlaxWeb.Issue
+
+  setup :verify_on_exit!
+
+  test "ignores non relevant events", _ do
+    event1 = %{"subtype" => "bot_message"}
+    event2 = %{"subtype" => "message_changed"}
+    event3 = %{"type" => "not_a_message"}
+    event4 = %{"bot_id" => "beep123"}
+
+    assert Issue.handle_event(event1) == nil
+    assert Issue.handle_event(event2) == nil
+    assert Issue.handle_event(event3) == nil
+    assert Issue.handle_event(event4) == nil
+  end
+
+  test "test regex", _ do
```

Comment:
Could this test name explain more about what the goal of using regex here is / what pattern we're trying to match and why?